### PR TITLE
Cumulus 1156

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** 
- Adds unit test scripts/configuration to bamboo deployment. 
- Adds updated compose stack that utilizes internal network to run tests instead of the deploy environment

For further context/links/etc look at : https://bugs.earthdata.nasa.gov/browse/CUMULUS-1156

### CODE REVIEW

File: bamboo/bootstrap-sftp.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+echo 'user:password' | chpasswd
+
+sed -i 's/Port 22/Port 2222/g' /etc/ssh/sshd_config
+chmod 600 /etc/authorized_keys/user
+/usr/sbin/sshd -D -f /etc/ssh/sshd_config
+# /usr/sbin/sshd -D -ddd -f /etc/ssh/sshd_config
```
Comment (Jkovarik) [inline bamboo/bootstrap-sftp.sh]: Remove this

**Comment by Jkovarik** [comment]
Updated configuration such that container doesn't need to expose ports to agent environment

File: bamboo/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    network_mode: "service:build_env"
+    environment:
+      ES_JAVA_OPTS: "-Xms750m -Xmx750m"
+  localstack:
+    image: localstack/localstack:0.8.7
+    network_mode: "service:build_env"
+    environment:
+      SERVICES: 'kinesis,lambda,s3,sns,sqs,dynamodb,dynamodbstreams'
+  build_env:
+    image: jlkovarik/cumulus_build_env:1
```
Comment (yjpa7145) [inline bamboo/docker-compose.yml]: Is there somewhere more permanent that we should be storing this image?

Comment (Jkovarik) [reply bamboo/docker-compose.yml]: @yjpa7145  yes - I need to work that out with Jon/Venku.  We'll either put it out on cumuluss, *or* just utilize the internal docker image repo.

**Pull request merged**